configure.ac

```diff
@@ -35,10 +35,6 @@ AC_CONFIG_SRCDIR(lib/shmem.cpp)
 AC_REVISION([$Revision$])
 REV=`echo '$Revision$' | awk "{print $2}"`
 RDATE=`date '+%Y.%m.%d'`
-if test -d .svn ; then
```

Comment:
Is this full block is really necessary?
https://github.com/BOINC/boinc/blob/ec2f4ee669e2b43bbfea41a763ab7b4c5e83d7d8/configure.ac#L35-L43
I'd delete it completely

Reply:
Why? Even without SVN `REV` and `RDATE` are set. Sure, if those two aren't used anywhere then the block can as well go.

Reply:
As far as I see, these two variables are not used anymore

Reply:
You sure? Aren't they used in every `Makefile.in`?

Reply:
They seem to be used in some [old `*.in` files](https://github.com/BOINC/boinc/search?q=RDATE&unscoped_q=RDATE). Are those still used/needed. Shouldn't they be generated dynamically anyway (via `automake`)?

Reply:
Search shows no usages:
![image](https://user-images.githubusercontent.com/3234209/76754183-f958a600-6781-11ea-80ee-41a595575dfc.png)
![image](https://user-images.githubusercontent.com/3234209/76754226-0aa1b280-6782-11ea-9a17-eac403c70361.png)

Reply:
Hm, files you mentioned just wa not shown in my search.
I'm not sure whether these files are still used

Reply:
Yep, GitHub's search has proven to be unreliable over the past weeks 😒

Reply:
Yep, GitHub's search has proven to be unreliable over the past weeks 😒